Code-4-Community/frontend-scaffold
# As.create navbar

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/677600458/pulses/785273678)

This is for the navigation bar of the LLB frontend. This will allow users to navigate to different parts of the website, for example a page to see their events and a page to update their profile.

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers. 
This required using an appropriate AntDesign component. For this, I decided to go with the Grid Component. Within a row, there are several columns, holding buttons that link to different pages. Also, the authentication state will be received (currently just using a boolean variable that can be turned on/off) to dynamically render the name of the user, or login/signup buttons if the user has not logged in or signed up yet.

## Screenshots

![image](https://user-images.githubusercontent.com/46979800/97246131-fcfde200-17d2-11eb-9c10-8cc47b20ba67.png)
![image](https://user-images.githubusercontent.com/46979800/97246174-130ba280-17d3-11eb-8852-48d1fafef83f.png)
![image](https://user-images.githubusercontent.com/46979800/97246261-4bab7c00-17d3-11eb-80b0-97dff4db8985.png)


## Verification Steps

Manual verification has been done so far, making sure that the correct information is being displayed upon a button click. Also, interacting with the navigation bar locally helped me confirm that the dropdown is being properly rendered upon a button hover.
